Forge.TreeWalker.UnitTests/Forge.TreeWalker.UnitTests.csproj

```diff
@@ -82,6 +83,12 @@
     <None Include="test\ExampleSchemas\TestEvaluateInputTypeSchema.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="test\InvalidTestSchemas\InvalidTestSchema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="test\InvalidTestSchemas\InvalidTestSchemaErrorContent.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
   </ItemGroup>
   <ItemGroup>
```

Comment:
>ItemGroup [](http://example.com/codeflow?start=3&length=9)

I think something like this will work to get the rules file in the output folder.  
<ItemGroup>
    <None Include="..\Forge.TreeWalker\contracts\ForgeSchemaValidationRules.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
  </ItemGroup>